pyrene/src/components/Loader/Loader.tsx

```diff
@@ -37,13 +37,13 @@ const Loader: React.FC<LoaderProps> = ({
   <>
     {type === 'standalone'
       && (
-        <div styleName={clsx('canvas', { [`size-${size}`]: true })}>
-          <div styleName={clsx('standaloneLoader', { [`styling-${styling}`]: true }, { [`size-${size}`]: true })} />
+        <div className={clsx(styles.canvas, { [styles[`size-${size}`]]: true })}>
```

Comment:
Please do the following when the `class` will be added anyway, means always `true`.

```
<div className={clsx(styles.canvas, styles[`size-${size}`])}>
```